hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/TrashOzoneFileSystem.java

```diff
@@ -283,7 +285,9 @@ public FileStatus getFileStatus(Path path) throws IOException {
   }
 
   private OmKeyArgs constructOmKeyArgs(Path path) {
-    OFSPath ofsPath = new OFSPath(path);
+    /*OzoneConfiguration conf = getConf() == null ? new OzoneConfiguration() :
+        OzoneConfiguration.of(getConf());*/
```

Comment:
Is this intended to be used later at some point? If not, we should remove it.

Reply:
Unneeded.  Thanks.  Removed.